pkg/opssight/ctl_opssight.go

```diff
@@ -279,6 +279,33 @@ func (ctl *Ctl) AddSpecFlags(cmd *cobra.Command, master bool) {
 	cmd.Flags().IntVar(&ctl.BlackduckMaxCount, "blackduck-max-count", ctl.BlackduckMaxCount, "Maximum number of Black Ducks that can be created")
 }
 
+var nSpecFlagCmd cobra.Command // command for getting flags for this spec
+var specFlags *pflag.FlagSet   // set of flags for this spec
+var numSpecFlagsChanged int    // number of flags that were set for the spec
+
+// NSpecFlag returns the number of spec flags that were set
+func (ctl *Ctl) NSpecFlag(flagset *pflag.FlagSet) int {
```

Comment:
Might as well make the function name descriptive, like NumNewFlags or something.